README.md

```diff
@@ -1 +1,38 @@
-# hello
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# HelloMicroverse
+
+The main goals of this project are:
+The main goals of this project to understand these things: 
+-How to create branches with different base branches.
+1-->To deal with version control.
+2-->Follow GitHub flow.
+3-->Open pull request.
+4-->Write descriptive to understand commit messages.
+5-->Write short, detailed and easy to understand descriptions of pull requests.
+6-->Demonstrate an ability to submit a project for code review.
+
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+
+## Authors
+
```

Comment:
- [ ] As pointed out by the previous reviewer, it is a requirement for this project, and also in future projects in microverse, endeavor to add it. 👉 Kindly make sure to add a good Pull Request description. This description is a brief outline of things you implemented in that pull request. 😉

- [ ] Good job adding this README file as requested by the previous reviewer, your repository is now professional. But there is a missing section, the "Getting Started" section. This section is important in giving intending users of your repo a proper guide on how to use it. Kindly add this section but have non-technical users (like HR persons) in mind.